hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockOutputStreamEntry.java

```diff
@@ -267,7 +269,11 @@ void executePutBlock() throws IOException {
       if (stream == null) {
         continue;
       }
-      stream.executePutBlock(false, true);
+      try {
+        stream.executePutBlock(false, true);
+      } catch (Exception e) {
+        stream.setIoException(e);
```

Comment:
If we are not throwing the exception here, that means we are effectively hiding a putBlock failure from the code that uses this code path.
I see two places, one is checkAndWriteParityCells, in the ECKeyOutputStream class, which first use the executePutBlock during handleParityWrites, where it checks the failed streams, but handleStripeFailure does not check for errors after calling the executePutBlock. So if the error happens during stripe failure handling we do not throw the exception anymore. When it will be checked? As in stripe failure handling after the executePutBlock, we just release the ec buffers as I see. Do I miss something?

Reply:
I see, that sounds good, thank you for the explanation, and for pointing out the other JIRA to handle this, I am ok to resolve this later on there.